sunbeam-python/sunbeam/commands/terraform.py

```diff
@@ -168,7 +168,7 @@ def init(self) -> None:
             LOG.warning(e.stderr)
             raise TerraformException(str(e))
 
-    def apply(self):
+    def apply(self, extra_args: list = []):
```

Comment:
```suggestion
    def apply(self, extra_args: list | None = None):
```
Please don't use mutable containers in args (especially in functions that might be called multiple times during an interpreter's lifetime)

Reply:
Fixed and pushed the change